src/barcoKeyboard/keyboardOpener.ts

```diff
@@ -0,0 +1,21 @@
+import { IpcRenderer } from 'electron'
+import { VirtualKeyboardEvent } from './events'
+
+// tslint:disable-next-line: function-name
+export function KeyboardOpener<T extends new (...args: any[]) => {}>(yo: T) {
```

Comment:
Is it missing a `=` to make an arrow function `=>` or is it some syntax i've never seen ?
`{}>(yo: T)`

Reply:
I'll delete this file, it was an attempt that was unneeded in the end